### ISSUE TITLE
$es->indices()->existsAlias() generating 404 warnings in log file

### ISSUE THREAD

**Issue by pmishev** [elasticsearch/elasticsearch-php#163]
I am trying to check if a certain alias exists (for any index).

The request is:
```
$es->indices()->existsAlias(['name' => $alias]);
```
In the log file I'm getting:
```
log.WARNING: Response ["Client error response\n[status code] 404\n[reason phrase] Not Found\n[url] http://localhost:9200/%2A/_alias/non_existent_alias"] 
```
It's not right to generate warnings for that.

The problem is complex I think because in the console I get:
```
$ curl -XGET 'localhost:9200/_alias/non_existent_alias'
{"error":"alias [non_exist] missing","status":404}

$ curl -XGET 'localhost:9200/%2A/_alias/non_existent_alias'
{}
```

...which I believe is a bug in Elasticsearch itself, because not passing the optional {index} param should be treated the same as if '*' wildcard is passed. Apparently this is not happening.

I am submitting it as an issue here though, because if I do:
```
$es->indices()->existsAlias(['index'=>'*','name' => $alias]);
```
I get:
```
log.WARNING: Response ["Client error response\n[status code] 404\n[reason phrase] Not Found\n[url] http://localhost:9200/%2A/_alias/non_existent_alias"] 
```
...which is not in line with the raw curl request as shown above.
So basically, I can't even see a workaround in order to suppress these warnings.
